utils/discounts/registration.ts

```diff
@@ -0,0 +1,24 @@
+// Discount details to fill in case of a duplicate of this page
```

Comment:
`registration.ts` is too vague, I would propose `evergreen`, evergeen mean we'll add here all discounts that don't have time limits or expiries.